### ISSUE TITLE
李宏毅机器学习-hw3-CNN-总结 · 核子的Blog

### ISSUE THREAD

**Issue by hezj-opt** [hezj-opt/comments#9]
https://hezj-opt.github.io/Deep-Learning/%E6%9D%8E%E5%AE%8F%E6%AF%85%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/%E6%9D%8E%E5%AE%8F%E6%AF%85%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0-hw3-CNN-%E6%80%BB%E7%BB%93/ 

 任务介绍
训练结果
训练方法

Data augmentation
模型设计
Loss function的选择与调参
Learning rate调整方案
Test time augmentation

进一步提升的可能

数据增强可能可以尝试mix up
模型设计可能可以继续尝试
Learning
